Brokerages/Alpaca/Markets/AlpacaTradingClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * The official C# API client for alpaca brokerage
+ * Sourced from: https://github.com/alpacahq/alpaca-trade-api-csharp/tree/v3.5.5
+*/
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+
+namespace QuantConnect.Brokerages.Alpaca.Markets
+{
+    /// <summary>
+    /// Provides unified type-safe access for Alpaca Trading API via HTTP/REST.
+    /// </summary>
+    public sealed partial class AlpacaTradingClient : IDisposable
+    {
+        // TODO: olegra - use built-in HttpMethod.Patch property in .NET Standard 2.1
+        private static readonly HttpMethod _httpMethodPatch = new HttpMethod("PATCH");
+
+        private readonly HttpClient _httpClient = new HttpClient();
+
+        private readonly IThrottler _alpacaRestApiThrottler;
+
+        /// <summary>
+        /// Creates new instance of <see cref="AlpacaTradingClient"/> object.
+        /// </summary>
+        /// <param name="configuration">Configuration parameters object.</param>
+        public AlpacaTradingClient(
+            AlpacaTradingClientConfiguration configuration)
+        {
+            configuration
+                .EnsureNotNull(nameof(configuration))
+                .EnsureIsValid();
+
+            _alpacaRestApiThrottler = configuration.ThrottleParameters.GetThrottler();
+
+            _httpClient.AddAuthenticationHeaders(configuration.SecurityId);
+
+            _httpClient.DefaultRequestHeaders.Accept
+                .Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            _httpClient.BaseAddress = configuration.ApiEndpoint
+                .AddApiVersionNumberSafe(configuration.ApiVersion);
+            _httpClient.SetSecurityProtocol();
```

Comment:
Seems this line will not be compiled unless we `#define NET45` ?
https://github.com/QuantConnect/Lean/pull/4228/files#diff-d6fdf1446f3e7eab9248163398e45ec8R141

Reply:
I define the NET45 constant at the top of the file on line 1

Reply:
It's needed in `Brokerages/Alpaca/Markets/AlpacaTradingClient.cs`